SPEC_PC_N/resource.h

```diff
@@ -2,8 +2,8 @@
 // fichier Include Microsoft Visual C++.
 // Utilisé par SETUP.RC
 //
-#define SUBLANG_ENGLISH_US              0x01
-#define LANG_ENGLISH                    0x09
+#define SUBLANG_ENGLISH_US              1
+#define LANG_ENGLISH                    9
```

Comment:
Could these two be mixed? It seems Odd that the `LANG_ENGLISH` would have 8 and 1 set, but `SUBLANG_ENGLISH_US` would only have bit 1

If true then it's an oddity, but if it is a mistake then `LANG_ENGLISH` would be 1 and 8 would set the variant, but both would need to be set so `SUBLANG_ENGLISH_US` = `LANG_ENGLISH | SUBLANG_US` maybe (does assume rather simple layout and infer particular implementation)